code/modules/client/client_procs.dm

```diff
@@ -380,8 +380,6 @@
 		if(src)
 			winset(src, null, "command=\".configure graphics-hwmode off\"")
 			winset(src, null, "command=\".configure graphics-hwmode on\"")
-			// fit_viewport here too as its all winset/get commands
-			fit_viewport() // required to prevent people from logging in for the first time and their screen being SUPER tiny
 
```

Comment:
```suggestion
			// fit_viewport here too as its all winset/get commands
			fit_viewport() // required to prevent people from logging in for the first time and their screen being SUPER tiny
			
```